libnmstate/nm/ipv6.py

```diff
@@ -133,6 +134,10 @@ def create_setting(config, base_con_profile):
 
     if is_dhcp or is_autoconf:
         _set_dynamic(setting_ip, is_dhcp, is_autoconf)
+        # NetworkManager will remove the virtual interface when DHCPv6 or
+        # IPv6-RA timeout, set them to infinity.
+        setting_ip.props.dhcp_timeout = INT32_MAX
+        setting_ip.props.ra_timeout = INT32_MAX
```

Comment:
Since this is part of 1.22.8 should we update the specfile to require 1.22.8 or later?

Reply:
If this is not available before, then we must. Or do I miss something here?
(another option is to do this on-best-effort or per NM version)